(DOCS-3542) Move service catalog to new section for GA (#14714)
DataDog/documentation: content/en/tracing/service_catalog/integrations.md

---
title: Using Integrations with Service Catalog
kind: documentation
further_reading:
- link: "/tracing/service_catalog/service_definition_api/"
  tag: "Documentation"
  text: "Service Definition API"
---

## PagerDuty integration

> You can add PagerDuty metadata to a service so that Service Catalog it displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service.

Author: ```suggestion
You can add PagerDuty metadata to a service so that Service Catalog displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service.
```

> You can add PagerDuty metadata to a service so that Service Catalog it displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service.

Reviewer: ```suggestion
You can add PagerDuty metadata to a service so that Service Catalog displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service.
```

### Setup

You can connect any service in your [PagerDuty Service Directory][1]. You can map one PagerDuty service for each service in Service Catalog.

1. If you have not already done so, set up the [Datadog PagerDuty integration][2].

> 2. Get your PagerDuty API access key as described at [https://support.pagerduty.com/docs/api-access-keys][3].

Author: ```suggestion
2. Get your PagerDuty API access key as described in their [API Access Key][3] documentation.
```
